### DISCUSSION TITLE
Gettint error message "Error: spawn EPERM" when running cypress e2e test

### DISCUSSION THREAD

**Discussion by Alok-Sha** [cypress-io/cypress#27449] [CI setup]
When running E2E test , I am getting below error logs : 

An unexpected error occurred while verifying the Cypress executable.

Please search Cypress documentation for possible solutions:

https://on.cypress.io

Check if there is a GitHub issue describing this crash:

https://github.com/cypress-io/cypress/issues

Consider opening a new issue.

----------

Error: spawn EPERM

----------

Platform: win32 (10.0.22000)
Cypress Version: 6.9.1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tmo@0.0.0 e2e: `nx e2e okreads-e2e`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the tmo@0.0.0 e2e script.

**Comment by ntl-lalain** [comment]
did you find the solution? facing the same issue rn. might be because of my antivirus but not sure
